great_expectations/dataset/base.py

```diff
@@ -1980,6 +1980,81 @@ def expect_column_values_to_match_json_schema(self,
 
     ##### Aggregate functions #####
 
+    def expect_column_parameterized_distribution_ks_test_p_value_to_be_greater_than(self,
+                                                                                    column, distribution,
+                                                                                    p_value=0.05, params=None,
+                                                                                    result_format=None,
+                                                                                    include_config=False,
+                                                                                    catch_exceptions=None, meta=None):
+        """
+        Expect the column values to be distributed similarly to the provided scipy distribution. \
+
+        This expectation compares the provided column to the specified continuous distribution with a parameteric \
+        Kolmogorov-Smirnov test. The K-S test compares the provided column to the cumulative density function (CDF) of \
+        the specified scipy distribution. If you don't know the desired distribution shape parameters, use the \
+        `ge.dataset.util.infer_distribution_parameters()` utility function to estimate them.
+
+        It returns 'success'=True if the p-value from the K-S test is greater than or equal to the provided p-value.
+
+        expect_column_parameterized_distribution_ks_test_p_value_to_be_greater_than is a \
+        :func:`column_aggregate_expectation <great_expectations.dataset.base.DataSet.column_aggregate_expectation>`.
+
+        Args:
+            column (str): \
+                The column name.
+            distribution (str): \
+                The scipy distribution name. See: https://docs.scipy.org/doc/scipy/reference/stats.html
+            p_value (float): \
+                The threshold p-value for a passing test. Default is 0.05
+            params (dict) : \
+                A dictionary of shape parameters that describe the disrtibution you want to test the data against.\
+                If not provided, the parameters will be attempted to be constructed using the column data and \
+                the specified distribution. Include key values specific to the distribution from the appropriate scipy \
+                distribution CDF function. \
+                See https://docs.scipy.org/doc/scipy/reference/stats.html#continuous-distributions
+
+        Other Parameters:
+            output_format (str or None): \
```

Comment:
Small tweak needed here: we renamed output_format to result_format in the new docs. Grab the "other parameters" section from an expectation now to update, please.

Reply:
Went ahead and copied over the boilerplate documentation from another expectation. It should match the current nomenclature.